### PR TITLE
[AKS] `az aks update`: Add `--ip-families` to support updating cluster network settings GA

### PR DESCRIPTION
**Pull request by camrynl**
**Related command**

`az aks update --ip-families ipv4,ipv6`

**Description**
Brings --ip-families usage to CLI GA
 
**Testing Guide**

**History Notes**

[AKS] `az aks update`: Add `--ip-families` to support updating cluster network

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py [hunk 1]
```diff
@@ -9387,6 +9387,49 @@ def test_aks_create_dualstack_with_default_network(self, resource_group, resourc
         # delete
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+    
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_update_dualstack(self, resource_group, resource_group_location):
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py:9393]: Please commit the corresponding recording file (find it from pipeline artifact if the live test passed).

**Review by FumingZhang — COMMENTED** [review]
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=108594698&view=results) to validate the change.

- test_aks_update_dualstack

File: src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py (hunk 1 above)
Comment (camrynl) [reply src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py:9393]: thanks, updated

**Comment by FumingZhang** [comment]
@camrynl please resolve the conflicts

**Review by FumingZhang — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**